src/components/Popup.js

```diff
@@ -28,6 +29,15 @@ export default class Popup {
     }
   }
 
+  setProcessingMessage(text) {
+    console.log(this._submitButton);
+    this._submitButton.textContent = text;
+  }
+
+  removeProcessingMessage(text) {
+    this._submitButton.textContent = text;
+  }
```

Comment:
Please, delete it from `Popup` because the image modal doesn't have a submit button